### PR TITLE
DTSBPS-445: springboot configtree setup

### PR DESCRIPTION
**Pull request by rishikrsharma**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/DTSBPS-445

### Change description ###
Adding springboot configtree feature - removing bootstrap yaml

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

**Review by jasonpaige — APPROVED** [review]
🚀

File: charts/ctsc-work-allocation/values.yaml [hunk 1]
```diff
@@ -4,19 +4,29 @@ java:
...
-        - wa-service-email-address
-        - wa-service-probate-email-address
-        - work-allocation-POSTGRES-PASS
-        - CTSC-WA-SMTP-USER
-        - CTSC-WA-SMTP-PASSWORD
-        - CTSC-S2S-SECRET
+        - name: AppInsightsInstrumentationKey
+          alias: ctsc.AppInsightsInstrumentationKey
+        - name: CTSC-IDAM-CLIENT-SECRET
+          alias: ctsc.CTSC-IDAM-CLIENT-SECRET
```
Comment (adusumillipraveen) [inline charts/ctsc-work-allocation/values.yaml:11]:
```suggestion
          alias: CTSC-IDAM-CLIENT-SECRET
```

File: charts/ctsc-work-allocation/values.yaml [hunk 2]
```diff
@@ -4,19 +4,29 @@ java:
...
-        - CTSC-SERVICE-USER-PASSWORD
-        - CTSC-SERVICEBUS-CONNECTION-STRING
-        - wa-service-email-address
-        - wa-service-probate-email-address
-        - work-allocation-POSTGRES-PASS
-        - CTSC-WA-SMTP-USER
-        - CTSC-WA-SMTP-PASSWORD
-        - CTSC-S2S-SECRET
+        - name: AppInsightsInstrumentationKey
+          alias: ctsc.AppInsightsInstrumentationKey
```
Comment (adusumillipraveen) [inline charts/ctsc-work-allocation/values.yaml:9]: I think it should be ?
```suggestion
          alias: AZURE_APPLICATIONINSIGHTS_INSTRUMENTATIONKEY
```

**Review by adusumillipraveen — COMMENTED** [review]
same for others? doesnt ur app use secrets in preview? to quickly test it ?

File: charts/ctsc-work-allocation/values.yaml (hunk 2 above)
Comment (rishikrsharma) [reply charts/ctsc-work-allocation/values.yaml]: applied change

File: charts/ctsc-work-allocation/values.yaml (hunk 1 above)
Comment (rishikrsharma) [reply charts/ctsc-work-allocation/values.yaml]: change has been made

**Review by adusumillipraveen — APPROVED** [review]
can u remove this, it has no value now
https://github.com/hmcts/ctsc-work-allocation/blob/869251e9c0f68e4268503ab79f1655211885d47c/charts/ctsc-work-allocation/values.preview.template.yaml#L4

**Pull request merged**